README.md

```diff
@@ -30,9 +30,8 @@ If you have questions or need help, please check out our documentation for a [li
 
 ## conda-forge dev meetings
 
-Our documentation contains a section with [minutes from previous dev meetings]([https://conda-forge.org/docs/minutes/00_intro.html]). These meetings occur every two weeks on Wednesday from 17:00-18:00 UTC.
-A link to the google calendar item can be found [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=Z2lraDk2a205cGUxdDkxYmNybXQxMGIxMGtfMjAxOTA3MjRUMTcwMDAwWiBzY29wYXR6QG0&tmsrc=scopatz%40gmail.com&scp=ALL).
+We hold biweekly meetings every second Wednesday from 17:00-18:00 (UTC). Feel free to stop by with the help of this [Google Calendar invite](https://calendar.google.com/calendar/u/0/r/eventedit/copy/Z2lraDk2a205cGUxdDkxYmNybXQxMGIxMGtfMjAxOTA3MjRUMTcwMDAwWiBzY29wYXR6QG0/c2hsb2thcHJpbmNlc3MxMDFAZ21haWwuY29t?scp=ALL&sf=true).
```

Comment:
Hm, I was trying to use this URL to see what a new user would see and... all I get is a blank Calendar page. I think we need this one instead:

https://calendar.google.com/event?action=TEMPLATE&tmeid=bTk5ZzBoMDEzaW11cmZiNWJnNmNkbThocDRfMjAyMjA3MTNUMTcwMDAwWiBqcm9kcmlndWV6QHF1YW5zaWdodC5jb20&tmsrc=jrodriguez%40quansight.com&scp=ALL